stats/test/platform-routes.test.js

```diff
@@ -362,6 +362,57 @@ describe('Platform Routes HTTP request handler', () => {
       )
     })
   })
+
+  describe('GET /participants/accumulative/daily', () => {
+    it('counts accumulative daily participants', async () => {
+      // 3 new participants, out of range
+      await givenDailyParticipants(
+        pgPools.evaluate,
+        '1999-01-01',
+        ['0x10', '0x20', '0x30']
+      )
+      // 3 new participants, 1 old participant -> 6
+      await givenDailyParticipants(
+        pgPools.evaluate,
+        '2000-01-01',
+        ['0x1', '0x2', '0x3', '0x10']
+      )
+      // 0 new participants, 2 old participants
+      await givenDailyParticipants(
+        pgPools.evaluate,
+        '2000-01-02',
+        ['0x1', '0x2']
+      )
+      // 1 new participant, 1 old participant -> 7
+      await givenDailyParticipants(
+        pgPools.evaluate,
+        '2000-01-03',
+        ['0x1', '0x4']
+      )
+      // 1 new participant, out of range
+      await givenDailyParticipants(
+        pgPools.evaluate,
+        '2000-01-04',
+        ['0x5']
+      )
+
+      const res = await fetch(
+        new URL('/participants/accumulative/daily?from=2000-01-01&to=2000-01-03', baseUrl), {
+          redirect: 'manual'
+        }
+      )
+      await assertResponseStatus(res, 200)
+      const daily = await res.json()
+      assert.deepStrictEqual(daily, [
+        { day: '2000-01-01', participants: 6 },
```

Comment:
I think this test covers the question I described in my comment above.

Here is the data we have:

```js
      // 3 new participants, out of range
      await givenDailyParticipants(
        pgPools.evaluate,
        '1999-01-01',
        ['0x10', '0x20', '0x30']
      )
      // 3 new participants, 1 old participant -> 6
      await givenDailyParticipants(
        pgPools.evaluate,
        '2000-01-01',
        ['0x1', '0x2', '0x3', '0x10']
      )
```

- On 1999-01-01, three participants joined the network - '0x10', '0x20', '0x30'
- On 2000-01-01, 0x10 was active, other two existing participants were not, and three new participants joined the network - '0x1', '0x2', '0x3'

Our API endpoint gives the number of participants that joined the network before or on 2000-01-01.

The correct answer is 6 ('0x10', '0x20', '0x30', '0x1', '0x2', '0x3'), and your test asserts the correct value 👍🏻 

Then 0x4 joins on 2000-01-03 and when we ask fro the number of participants that joined the network before or on 2000-01-03, we get back 7, which is again the correct answer 👍🏻